Fantasy-Quest/Assets/Scripts/Dialogue/SymonSpeaker.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using Spine.Unity;
+using UnityEngine;
+
+namespace Dialogue
+{
+    [RequireComponent(typeof(Collider2D))]
+    [RequireComponent(typeof(AudioSource))]
+    [AddComponentMenu("Scripts/Dialogue/Dialogue.SymonSpeaker")]
+    public class SymonSpeaker : DialogueSpeaker
+    {
+        [SerializeField]
+        private SkeletonAnimation skeletonAnimation;
+
+        private void Awake()
+        {
+            if (!SubtitlesViewGameObject.TryGetComponent(out SubtitlesView))
+            {
+                Debug.LogError(
+                    $"{SubtitlesView} is NULL\n{GetType()} callback in {gameObject.name}"
+                );
+            }
+
+            Voice = new Voice(GetComponent<AudioSource>());
+        }
+
+        public override void Stop()
+        {
```

Comment:
use base.Stop() instead of repeating code

Reply:
There's new line of code(34), so it was needed to use override

Reply:
My mistake, i get it now